django_ai_assistant/helpers/assistants.py

```diff
@@ -577,7 +701,7 @@ def as_tool(self, description: str) -> BaseTool:
         Returns:
             BaseTool: A tool that runs the assistant. The tool name is this assistant's id.
         """
-        return Tool.from_function(
+        return StructuredTool.from_function(
```

Comment:
Is this change necessary?

Reply:
Yes, that's due to the update to `langchain-core==0.2.38` required to use `langgraph`. Running with `Tool` gives an error due to tool parameters not aligning and the error itself recommends using `StructuredTool`